shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -429,10 +410,10 @@ - (void)setViewController:(FlutterViewController*)viewController {
   self.iosPlatformView->SetOwnerViewController(_viewController);
   [self maybeSetupPlatformViewChannels];
   [self updateDisplays];
-  _textInputPlugin.get().viewController = viewController;
+  self.textInputPlugin.viewController = viewController;
 
   if (viewController) {
-    __block FlutterEngine* blockSelf = self;
+    __weak __block FlutterEngine* blockSelf = self;
```

Comment:
This is just preserving the existing weak pointer semantics.

Reply:
Nit: it's much more common to call this variable `weakSelf`, and that's also more descriptive, so I would suggest using that convention.

Reply:
Done.